exercises/binary-search/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "java"
+apply plugin: "eclipse"
+apply plugin: "idea"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  testCompile "junit:junit:4.12"
+  testCompile "org.assertj:assertj-core:3.2.0"
```

Comment:
In general, my preference (moving forward) would be for all new exercise implementations to rely on the default JUnit assertions. While I definitely think AssertJ assertions read better, it's another API surface that newcomers need to explore/understand to make sense of the tests we present.

@exercism/java thoughts on this? Is there somewhere we can capture standards such as the one I'm proposing for future reference? Thanks!